docs/conf.py

```diff
@@ -65,9 +65,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.1.0'
+version = '0.2.0'
 # The full version, including alpha/beta/rc tags.
-release = '0.1.0'
+release = '0.2.0'
```

Comment:
Oh yeah. I always forget about Sphinx versioning. I'll try to get that to pull automatically, too.